cmd/kubernetes-admission-controller/extractor/from_job.go

```diff
@@ -0,0 +1,25 @@
+package extractor
+
+import (
+	"encoding/json"
+
+	"k8s.io/api/admission/v1beta1"
+	batchV1 "k8s.io/api/batch/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Enforcing compliance with the Extractor type
+var _ Extractor = fromJob
+
+// fromDeployment returns the extracted object metadata and included v1.PodSpecs from the Deployment admission request.
```

Comment:
(I'm sorry, @zhill — I missed this in the first PR, but I should've caught it then.) The doc comment needs to match the function name in order to be useful in the Go tooling. In this case, to make the meaning clear, we'd also want to update mentions of "Deployments" elsewhere in the comment, too.

Here's what I'd suggest:

```suggestion
// fromJob returns the extracted object metadata and included v1.PodSpecs from the Job admission request.
```